### ISSUE TITLE
[question] How use facet with aggragation functions?

### ISSUE THREAD

**Issue by genby8** [manticoresoftware/manticoresearch-php#119]
Read all client documents. Looked at the implementation of "facet" in the code. Am I right in thinking that there is no such functionality? For example "MIN" or "MAX"?
It after all as not simply to create sql. Worry about escaping field names or something else.
Or is there still some way to use the same "searchclass" + SQL ?

**Comment by sanikolaev** [comment]
You are right. You have to use the SQL method for that https://github.com/manticoresoftware/manticoresearch-php/blob/master/docs/sql.md

**Comment by genby8** [comment]
Thanks for the quick response. It is a pity that it is impossible to use the wrapper.

**Comment by sanikolaev** [comment]
Right. Let's keep this issue open as a feature request. The point is that we'll have to implement min/max in the Manticore JSON first (which the clients use).

**Comment by sanikolaev** [comment]
Related forum topic https://forum.manticoresearch.com/t/using-min-with-manticoresearch-php-sdk/1426/2
